sdk/python/kfp/dsl/_ops_group.py

```diff
@@ -101,6 +101,14 @@ def remove_op_recursive(self, op):
     for sub_group in self.groups or []:
       sub_group.remove_op_recursive(op)
 
+
+class Parallelism(OpsGroup):
```

Comment:
I'm not sure this is sustainable: Imagine if we decided to add another option like node selector to a group of nodes. Then we'll have to add new OpsGroup class.

What if we named this class something neutral like `SubGraph`?

Reply:
Maybe it would be better to extend the graph component, and add the argument there instead? Since the Graph component annotation will create a subgraph.  What are your thoughts on making it more general? I think the name might be a bit confusing with the graph component.

Reply:
Let me know your thoughts @Ark-kun, I have done some updates testing how it would work if we put it in ```graph_component```

Reply:
I'm not sure extending the `@graph_component` decorator is a good idea. It's decorator, not context. It's supposed to be applied to functions.

It's OK to try to reuse the `Graph` class though.

Reply:
> I'm not sure this is sustainable: Imagine if we decided to add another option like node selector to a group of nodes. Then we'll have to add new OpsGroup class.
> 
> What if we named this class something neutral like `SubGraph`?

Maybe I actually misunderstood this initial comment, did you only mean that we should make a name change here and that we could keep the implementation @Ark-kun ? To have a more common functionallity? 

If we go with the `SubGraph`way should we also try to combine functionality as exithandling, to this class instead of having it as to separate? 

>I'm not sure extending the @graph_component decorator is a good idea. It's decorator, not context. It's supposed to be applied to functions.
>

My thoughts where that we could set the behaviour to be incapsulated based upon the fuctions, however I do prefer the original implementation and think it is more in line with the way the SDK works.